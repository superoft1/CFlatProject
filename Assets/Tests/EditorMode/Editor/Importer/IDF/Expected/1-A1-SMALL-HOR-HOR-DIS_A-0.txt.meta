fileFormatVersion: 2
guid: 2ba60093186b4c1489a7f52c6b659fc3
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
