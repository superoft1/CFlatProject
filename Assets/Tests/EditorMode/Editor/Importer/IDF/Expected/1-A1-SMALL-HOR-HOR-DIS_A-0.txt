Pipe, , (0.0, 0.0, 1.0), (-101.4, 10.8, 0.9), False
(-101.4, 10.8, 0.8)
(-101.4, 10.8, 0.9)
FreeVertex: 1
ConcentricPipingReducer, , (0.0, 0.0, 1.0), (-101.4, 10.8, 1.0), False
(-101.4, 10.8, 1.0)
(-101.4, 10.8, 0.9)
FreeVertex: 0
Elbow90, , (0.6, -0.8, 0.0), (-101.4, 10.8, 1.2), False
(-101.4, 10.8, 1.0)
(-101.5, 10.9, 1.2)
FreeVertex: 0
Pipe, , (-0.6, 0.8, 0.0), (-101.7, 11.1, 1.2), False
(-101.5, 10.9, 1.2)
(-101.8, 11.3, 1.2)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, 1.0), (-101.8, 11.3, 1.2), False
(-101.8, 11.3, 1.2)
(-101.8, 11.3, 1.2)
(-101.8, 11.3, 1.2)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-101.8, 11.3, 1.3), False
(-101.8, 11.3, 1.2)
(-101.8, 11.3, 1.3)
FreeVertex: 0
GlobeValve, , (0.0, 0.0, 1.0), (-101.8, 11.3, 1.3), False
(-101.8, 11.3, 1.3)
(-101.8, 11.3, 1.4)
FreeVertex: 0
PressureGauge, PG_102_11_10_1070_1E, (0.0, 0.0, 1.0), (-101.8, 11.3, 1.4), False
(-101.8, 11.3, 1.4)
(-101.8, 11.3, 1.4)
FreeVertex: 1
Pipe, , (-0.6, 0.8, 0.0), (-101.9, 11.3, 1.2), False
(-101.8, 11.3, 1.2)
(-101.9, 11.4, 1.2)
FreeVertex: 0
WeldNeckFlange, , (-0.6, 0.8, 0.0), (-101.9, 11.4, 1.2), False
(-102.0, 11.4, 1.2)
(-101.9, 11.4, 1.2)
FreeVertex: 0
CheckValve, , (-0.6, 0.8, 0.0), (-102.0, 11.5, 1.2), True
(-102.0, 11.5, 1.2)
(-102.0, 11.4, 1.2)
FreeVertex: 0
WeldNeckFlange, , (0.6, -0.8, 0.0), (-102.0, 11.5, 1.2), False
(-102.0, 11.5, 1.2)
(-102.1, 11.6, 1.2)
FreeVertex: 0
Pipe, , (-0.6, 0.8, 0.0), (-102.1, 11.6, 1.2), False
(-102.1, 11.6, 1.2)
(-102.2, 11.7, 1.2)
FreeVertex: 0
Pipe, , (-0.6, 0.8, 0.0), (-102.3, 11.8, 1.2), False
(-102.2, 11.7, 1.2)
(-102.4, 11.9, 1.2)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-102.4, 11.9, 1.2), False
(-102.4, 11.9, 1.2)
(-102.4, 11.9, 1.2)
(-102.4, 11.9, 1.1)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-102.4, 11.9, 1.0), False
(-102.4, 11.9, 1.1)
(-102.4, 11.9, 1.0)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-102.4, 11.9, 1.0), False
(-102.4, 11.9, 0.9)
(-102.4, 11.9, 1.0)
FreeVertex: 1
Pipe, , (-0.6, 0.8, 0.0), (-102.4, 12.0, 1.2), False
(-102.4, 11.9, 1.2)
(-102.4, 12.0, 1.2)
FreeVertex: 0
WeldNeckFlange, , (-0.6, 0.8, 0.0), (-102.5, 12.0, 1.2), False
(-102.5, 12.1, 1.2)
(-102.4, 12.0, 1.2)
FreeVertex: 0
Pipe, , (-0.6, 0.8, 0.0), (-102.5, 12.1, 1.2), False
(-102.5, 12.1, 1.2)
(-102.5, 12.1, 1.2)
FreeVertex: 0
GateValve, , (-0.6, 0.8, 0.0), (-102.6, 12.2, 1.2), False
(-102.6, 12.2, 1.2)
(-102.5, 12.1, 1.2)
FreeVertex: 0
WeldNeckFlange, , (0.6, -0.8, 0.0), (-102.6, 12.3, 1.2), False
(-102.6, 12.2, 1.2)
(-102.7, 12.3, 1.2)
FreeVertex: 0
Pipe, , (-0.6, 0.8, 0.0), (-102.7, 12.3, 1.2), False
(-102.7, 12.3, 1.2)
(-102.7, 12.4, 1.2)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-102.7, 12.4, 1.2), False
(-102.7, 12.4, 1.2)
(-102.7, 12.4, 1.2)
(-102.7, 12.4, 1.1)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-102.7, 12.4, 1.0), False
(-102.7, 12.4, 1.1)
(-102.7, 12.4, 1.0)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-102.7, 12.4, 1.0), False
(-102.7, 12.4, 0.9)
(-102.7, 12.4, 1.0)
FreeVertex: 1
Pipe, , (-0.6, 0.8, 0.0), (-102.8, 12.4, 1.2), False
(-102.7, 12.4, 1.2)
(-102.8, 12.4, 1.2)
FreeVertex: 0
Elbow90, , (0.0, 0.0, -1.0), (-102.9, 12.5, 1.2), False
(-102.8, 12.4, 1.2)
(-102.9, 12.5, 1.3)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-102.9, 12.5, 1.9), False
(-102.9, 12.5, 1.3)
(-102.9, 12.5, 2.6)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-102.9, 12.5, 2.7), False
(-102.9, 12.5, 2.6)
(-103.0, 12.5, 2.7)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-103.9, 12.5, 2.7), False
(-103.0, 12.5, 2.7)
(-104.7, 12.5, 2.7)
FreeVertex: 1
FreeVertexTotal: 5
